download-launch-agent.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Set up runner directory
+prefix=/opt/circleci
+sudo mkdir -p "${prefix}/workdir"
+[[ -z ${api_url+z} ]] && api_url="https://runner.circleci.com"
+[[ -z ${agent_version+z} ]] && agent_version=""
+[[ -z ${platform+z} ]] && echo "platform not defined" && exit 1
+[[ -z ${CIRCLECI_RUNNER_TOKEN+z} ]] && echo "CIRCLECI_RUNNER_TOKEN not defined" && exit 1
```

Comment:
Is this the runner token after creating a resource class or the personal api token from the UI?

Reply:
It's the resource class token.